src/lightstep-datasource/datasource.js

```diff
@@ -64,6 +64,16 @@ export class LightStepDatasource {
         params: query,
       });
 
+      response.then(result => {
+        if (result && result["data"]["data"]) {
```

Comment:
Suggestion: Use dot notation to shorten this code up a bit given that we're just using hard coded string keys.

Reply:
I got confused by the dot notation a little. I wasn't sure when I could and couldn't use it. We should chat IRL if you have a second.